### PR TITLE
More Cloud Functions

### PR DESCRIPTION
**Pull request by jmdobry**
Added two new Cloud Functions samples: SendGrid and OCR.

Made READMEs of various Cloud Functions samples more consistent.

### CODE REVIEW

**Comment by jmdobry** [comment]
@jsimonweb Can you review?

File: functions/ocr/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+          "RESULT_BUCKET": "[RESULT_BUCKET_NAME]",
+          "TRANSLATE_TOPIC": "[TRANSLATE_TOPIC_NAME]",
+          "TRANSLATE": true,
+          "TO_LANG": ["en", "fr", "es", "ja", "ru"]
+        }
+
+    * Replace `[YOUR_API_KEY]` with your Translate API key.
+    * Replace `[RESULT_TOPIC_NAME]` with a topic name used for saving results.
+    * Replace `[RESULT_BUCKET_NAME]` with a bucket name used for saving results.
+    * Replace `[TRANSLATE_TOPIC]` with a topic name used for translating results.
```
Comment (jsimonweb) [inline functions/ocr/README.md]: Update "Replace `[TRANSLATE_TOPIC]` ...

to be

Update `[TRANSLATE_TOPIC_NAME]` ...

Comment (jmdobry) [inline functions/ocr/README.md]: Done

**Comment by jsimonweb** [comment]
LGTM

**Pull request merged**